### ISSUE TITLE
mbp2018-bridge build fails

### ISSUE THREAD

**Issue by davidak** [NixOS/nixpkgs#120720]
**Describe the bug**

Upstream seem to have removed the release.

I hope they add it again.

```
builder for '/nix/store/i7rqvjkwwbfhrx8fnyip47fakaw9ld1l-source.drv' failed with exit code 1; last 8 log lines:
  
  trying https://github.com/MCMrARM/mbp2018-bridge-drv/archive/0.01.tar.gz
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
  100   132  100   132    0     0    545      0 --:--:-- --:--:-- --:--:--   545
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  curl: (22) The requested URL returned error: 404 
  error: cannot download source from any mirror
```

Upstream issue: https://github.com/MCMrARM/mbp2018-bridge-drv/issues/22

**To Reproduce**
1. ...
2. ...
3. ...

**Expected behavior**

**Screenshots**

**Additional context**

**Notify maintainers**
@hlolli 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by hlolli** [comment]
If the tag was removed, I guess we can always use the revision-sha?

**Comment by davidak** [comment]
sure. having actual releases with a known good state would be nice tho

**Comment by hlolli** [comment]
I will admit that recently I was forced to use actual darwin system, that said, I can still test if needed on mbp+nixos.

If this isn't blocking, then let's wait a bit.

**Comment by tutacat** [comment]
self built binary?
or master head? -> https://github.com/MCMrARM/mbp2018-bridge-drv/archive/refs/heads/master.tar.gz

**Comment by hlolli** [comment]
> self built binary? or master head? -> https://github.com/MCMrARM/mbp2018-bridge-drv/archive/refs/heads/master.tar.gz

uh?

**Comment by Redecorating** [comment]
It also doesn't have tags, but this fork has been updated with some bug fixes and the module renamed from "bce" to "apple-bce", and might be a better one to package. https://github.com/t2linux/apple-bce-drv

Also notable is this one that has the same bigfixes + the ability to disable audio https://github.com/t2linux/nixos-overlay/blob/master/packages/kernel-modules/apple-bce/default.nix

**Comment by Artturin** [comment]
fixed in e661b70903d7a4021b7d5b2268183978541ec6c6
